contents/blog/session-replay-pricing.md

```diff
@@ -0,0 +1,91 @@
+> - We’re now the cheapest session replay tool among competitive products.
+
+And this is just Part 1...
+
+## How will this impact me?
+
+It will vary on usage, but a customer generating 25k recorded sessions a month will now pay $85 a month, down from $430 a month. That's a huge $4,140 annual saving. You can [calculate your new price](/pricing) on the pricing page.
+
+![new pricing](https://res.cloudinary.com/dmukukwp6/image/upload/new_pricing_7e0af9ddc4.jpg)
+
+This makes using PostHog cheaper than any competing tool with similar features that publishes its pricing. Here’s a breakdown of costs at different tiers:
+
+| &nbsp;                   | **Old price** | **New price** | **Change**   |
+|------------------------|---------------|---------------|--------------|
+| **5,000 recordings**   | $0            | $0            | No change    |
+| **25,000 recordings**  | $430          | $85           | 5.1x cheaper |
+| **50,000 recordings**  | $505          | $173          | 2.1x cheaper |
+| **100,000 recordings** | $640          | $273          | 2.3x cheaper |
+| **250,000 recordings** | $1,025        | $543          | 1.9x cheaper |
+| **500,000 recordings** | $1,650        | $968          | 1.7x cheaper |
+
+Sadly, many tools don’t publish pricing. That’s their choice, but we’re pretty confident it’s not because they’re secretly really good value. How else are they going to justify those huge outbound sales teams, eh?
+
+We’re not just cheaper than replay tools designed for developers, either. Anyone using Hotjar for their website or web app can also save by switching to PostHog.
+
+![posthog-vs-hotjar](https://res.cloudinary.com/dmukukwp6/image/upload/hotjar_ace01c5cb7.jpg)
+
+And, unlike Hotjar, we support [session replays on mobile](/docs/session-replay/mobile) with our open beta for [iOS](/docs/libraries/ios) and [Android](/docs/libraries/android) SDKs – yes, [we are working on React Native](https://github.com/PostHog/posthog/issues/13269), folks, but feel free to vote for it on [our public roadmap](/roadmap) anyway.
+
+As before, every user gets 5,000 free recordings every month, too. Free stuff isn’t limited to the free tier.
+
+## Why are you doing this?
+
+Because our [pricing principles](/handbook/engineering/feature-pricing) dictate:
+
+1. We should charge as little as possible while remaining margin positive. We don’t believe in loss-leaders, or squeezing the maximum out of people who are willing to pay for our product.
+
+2. The true value of PostHog comes from using all our products together. We make money by onboarding customers onto more of them, not maximizing the profitability of individual tools.
+
+The whole “loss-leader” thing is important, btw. Loss-leaders are bad for customers and companies. 
+
+No one enjoys surprise price increases when a company decides it wants to make money now, or seeing products sunsetted because they’re not sustainable anymore.
```

Comment:
```suggestion
The whole “loss-leader” thing is important, btw. Loss-leaders are bad for customers and companies. No one enjoys surprise price increases when a company decides it wants to make money now, or seeing products sunsetted because they’re not sustainable anymore.
```